apps/remix/app/components/general/template/template-edit-form.tsx

```diff
@@ -165,6 +166,14 @@ export const TemplateEditForm = ({
       await Promise.all([
         updateTemplateSettings({
           templateId: template.id,
+          data: {
```

Comment:
it overrides the attachments and sets them to an empty array when adding the template placeholders, hence why this bit of code.